ansible/roles/test/files/tools/loganalyzer/loganalyzer.py

```diff
@@ -124,6 +126,7 @@ def place_marker_to_file(self, log_file, marker):
             self.print_diagnostic_message('Log file {} not found. Skip adding marker.'.format(log_file))
         self.print_diagnostic_message('log file:{}, place marker {}'.format(log_file, marker))
         with open(log_file, 'a') as file:
+            file.write(datetime.now().strftime("%b %d %H:%M:%S.%f") + ' ')
```

Comment:
Why this?

Reply:
It seems that loganalyzer has some logic like skip the lines that are not in the format it likes. This change aims to make the additional start string consistent with regular log messages. Loganalyzer would complain about unable to find the start message without this change.

Reply:
Ah! I think it is better if we fix the marker finding logic, instead of adding a patch here. Below lines are writing marker and new-line; do you know why LA skips the line that it added here?

Reply:
I believe the reason is that it tries to extract a timestamp to find out the earliest message. If there is no timestamp in the front, it would get some out of index errors. I believe adding a timestamp in front of the message could be useful in terms of the timing, and it would also align with other messages and the start message adding of syslog. So my preference is still to add the timestamp.
https://github.com/Azure/sonic-mgmt/blob/9585036c8ace56bf4dba0f6444c854f838d5a8a3/ansible/library/extract_log.py#L199